ci/jenkins/jobs/macros.yaml

```diff
@@ -231,6 +231,8 @@
           #!/bin/bash
           set -ex
           DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
+          # Delete all kind clusters present from more than 135 mins
```

Comment:
Why 135 minutes? This is oddly specific, can the comment clarify this choice?

Reply:
we have set 135 minutes timeout for both conformance and NetworkPolicy tests in the jobs because sometime NetworkPolicy takes more time to finish its executions( It depends on how many test cases are running and the time of each test cases ) although we can choose 120m also but for the safer side we have set 135m just like other ci jobs in project-lab.yaml, for example "rancher-networkpolicy".
 
so in case if tests gets stuck during their execution then it will wait for 135m and run will aborted automatically, and in aborted cases clusters became useless and we need to delete that in next run before creating new one. not sure if this clarifies your question?

Reply:
we have set 135 minutes timeout for both conformance and NetworkPolicy tests in the jobs because sometime NetworkPolicy takes more time to finish its executions( It depends on how many test cases are running and the time of each test cases ) although we can choose 120m also but just to be on the safe side we have set 135m just like other ci jobs in project-lab.yaml, for example "rancher-networkpolicy".

so in case if tests gets stuck during their execution then it will wait for 135m and run will aborted automatically, and in aborted cases clusters became useless and we need to delete that in next run before creating new one. not sure if this clarifies your question?

Reply:
the relevant information should be added top the comment